CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 * [CHANGE] Available command-line flags are printed to stdout, and only when requested via `-help`. Using invalid flag no longer causes printing of all available flags. #2691
 * [CHANGE] Experimental Memberlist ring: randomize gossip node names to avoid conflicts when running multiple clients on the same host, or reusing host names (eg. pods in statefulset). Node name randomization can be disabled by using `-memberlist.randomize-node-name=false`. #2715
 * [CHANGE] Memberlist KV client is no longer considered experimental. #2725
+* [CHANGE] Change target flag for purger from `data-purger` to `purger` and make delete request cancellation duration configurable. #2760
```

Comment:
Can you clarify where this happened?
Not, for instance, here: https://github.com/cortexproject/cortex/pull/2760/files#diff-596d5408728e4813da55bfbc5a171d3dR63

Reply:
Sorry, my bad. I just renamed the components and missed changing the flag. Will open a PR to fix this.